### ISSUE TITLE
Seeking in a stream

### ISSUE THREAD

**Issue by jagjordi** [mopidy/mopidy#2135]
Is it possible to seek (rewind back) on a stream?

For example, I have this stream: https://shoutcast.ccma.cat/ccma/catalunyaradioHD.mp3

If I play it using VLC I can rewind up to 2 hours. Is it possible to do that in mopidy? And if so, is it possible to do without the web interface? Such as websocket or MPD, home assistant integration, etc...

Thanks

**Comment by kingosticks** [comment]
I don't believe we support that. You can only rewind up to the point you started playing.

Note,  this is a bug tracker. Please ask questions at https://discourse.mopidy.com/.
